Gemfile

```diff
@@ -54,6 +54,3 @@ group :development do
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
-
-# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
```

Comment:
There is, actually, harm in leaving it there. @AkivaGreen had a problem installing this gem so @werebus and I just told him to remove it. We could either spend time troubleshooting why a gem used solely for windows is malfunctioning, or just take it out.